.github/workflows/android-perf.yml

```diff
@@ -229,10 +229,9 @@ jobs:
 
         PYTHON_EXECUTABLE=python bash .ci/scripts/setup-qnn-deps.sh
         PYTHON_EXECUTABLE=python bash .ci/scripts/build-qnn-sdk.sh
-        export EXECUTORCH_BUILD_QNN=ON
 
         export ANDROID_ABIS="arm64-v8a"
-        bash build/build_android_llm_demo.sh ${ARTIFACTS_DIR_NAME}
+        PYTHON_EXECUTABLE=python EXECUTORCH_BUILD_QNN=ON QNN_SDK_ROOT=/tmp/qnn/2.25.0.240728 bash build/build_android_llm_demo.sh ${ARTIFACTS_DIR_NAME}
```

Comment:
Isn't QNN_SDK_ROOT set already in setup-qnn-deps.sh? If it's overriding it to a different version, we must ensure that version is downloaded and configured correctly

Reply:
https://github.com/pytorch/executorch/blob/main/.ci/scripts/build-qnn-sdk.sh#L14 fyi